components/expenses/CreateExpenseForm.js

```diff
@@ -164,6 +184,30 @@ class CreateExpenseForm extends React.Component {
     this.props.onChange && this.props.onChange(expense);
   }
 
+  handlePayoutChange(attr, value) {
+    let expense;
+    if (value === 'banktransfer') {
+      const bankAccount = this.props.LoggedInUser.collective.payoutMethods.find(pm => pm.type === 'BANK_ACCOUNT');
+      expense = {
+        ...this.state.expense,
```

Comment:
When using the current state to build a `setState`, one should ideally use a function. See https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/no-access-state-in-setstate.md